rfq/manager.go

```diff
@@ -288,6 +297,16 @@ func (m *Manager) handleIncomingMessage(incomingMsg rfqmsg.IncomingMsg) error {
 				"request: %w", err)
 		}
 
+	case *rfqmsg.SellAccept:
+		// TODO(ffranr): The stream handler should ensure that the
```

Comment:
What additional information do we need to implement this TODO? Any reason not to do it in this PR?

Reply:
In this PR I'm just trying to add functionality for the asset sell payment initiation to match what we already have for the asset buy payment initiation. The functionality that this TODO refers to is also missing for the incoming `rfqmsg.BuyAccept` case.

I can add this functionality in a small future PR.

Reply:
Sounds good to me.